db/migrate/20160514215652_merge_first_name_and_last_name_on_users.rb

```diff
@@ -0,0 +1,8 @@
+class MergeFirstNameAndLastNameOnUsers < ActiveRecord::Migration
```

Comment:
I was aware of that, but I didn't think it was worth the trouble since the app isn't running yet. The columns are created in this PR as well so the only person who should be affected is me.